console/serve.go

```diff
@@ -27,12 +28,17 @@ func (s *ServeCommand) Run() lib.CommandRunner {
 		route *routes.Routes,
 		logger lib.Logger,
 		database infrastructure.Database,
+		migrations infrastructure.Migrations,
+		seeds seeds.Seeds,
 	) {
 		logger.Info(`+-----------------------+`)
 		logger.Info(`| GO CLEAN ARCHITECTURE |`)
 		logger.Info(`+-----------------------+`)
+		migrations.Migrate()
```

Comment:
@Denes-cilwal 
I think migrations run automatically after the change in #55. Can you check once?

Reply:
Or, might be the PR with Unit tests.